backend/engine/plugins/checkov/main.py

```diff
@@ -116,8 +149,9 @@ def run_checkov(
 
     stderr = docker_client.containers.run(
         CHECKOV_IMG_REF,
+        name=container_name,
         command=checkov_command,
-        remove=True,
+        auto_remove=True,
```

Comment:
`auto_remove` is preferred since the Docker daemon will handle removing the stopped container on exit.  `remove` relies on the client to remove the container, which it won't have a chance to do if the plugin is aborting.